lib/.gitignore

```diff
@@ -1,6 +1,14 @@
 autoload.php
 composer.lock
 composer/
+marienfressinaud/lib_opml/.git/
```

Comment:
😱 😉

Reply:
Ah ah, I would prefer to have something like `libopml`, but I created my account on Packagist with my name, and I don't if and how I can change that without creating a different account :sweat_smile:

Reply:
Is the `.git` folder coming with composer? :o

Reply:
Is the `.git` folder coming with composer ? :o

(previous comment removed, still fighting with my different GH accounts :sweat_smile:)

Reply:
I just added a `.gitattributes` file, it should be better with the next version :) https://framagit.org/marienfressinaud/lib_opml/-/blob/main/.gitattributes